src/Form/Type/SearchFacetsType.php

```diff
@@ -13,7 +13,6 @@
 namespace BitBag\SyliusElasticsearchPlugin\Form\Type;
 
 use BitBag\SyliusElasticsearchPlugin\Facet\RegistryInterface;
-use BitBag\SyliusElasticsearchPlugin\Model\SearchFacets;
```

Comment:
if it's an only place where we used `BitBag\SyliusElasticsearchPlugin\Model\SearchFacets` then maybe it could be safely removed?

Reply:
There are 2 places where we use it. For now let's focus on making it work since I'm going to refactor more logic in another PR.